http_calls.py

```diff
@@ -91,6 +100,7 @@ class RestTools(HttpCall):
     http_headers = {'Content-Type': 'application/json', 'Accept': 'application/json'}
 
     def getRequest(self, url, data=None, headers={}):
+        headers.update(g_headers)
```

Comment:
I think local request headers should override global headers for a particular request.